experimenter/experimenter/experiments/tests/test_models.py

```diff
@@ -814,11 +814,7 @@ def test_targeting_with_locales_and_countries_desktop(self):
         )
         self.assertEqual(
             experiment.targeting,
-            (
-                "(os.isMac) "
-                "&& (locale in ['en-CA', 'en-US']) "
-                "&& (region in ['CA', 'US'])"
-            ),
+            ("(os.isMac) && (locale in ['en-CA', 'en-US']) && (region in ['CA', 'US'])"),
```

Comment:
Not a fan of it reflowing strings across multiple lines :(

Reply:
Yeah it's definitely more aggressive about strings than black is